frontend/src/features/env/mutations.ts

```diff
@@ -42,18 +41,28 @@ export const useEnvMutations = (
     },
   })
 
-  const submitSwitchEnvFormFeedbackMutation = useMutation(
-    (args: SwitchEnvFeedbackFormBodyDto) => {
-      return submitSwitchEnvFormFeedback({
-        formInputs: args,
-        feedbackForm: feedbackForm,
-      })
-    },
-  )
-
   return {
     publicSwitchEnvMutation,
     adminSwitchEnvMutation,
-    submitSwitchEnvFormFeedbackMutation,
   }
 }
+
+export const useAdminFeedbackMutation = () => {
+  const { data: adminFeedbackForm } = useAdminFeedbackFormView()
+  return useMutation((args: AdminFeedbackFormDto) => {
+    return submitSwitchEnvFormFeedback({
+      formInputs: args,
+      feedbackForm: adminFeedbackForm,
```

Comment:
seems like a dangerous regression here, what if the feedback form query has not resolved yet? may need some error handling.

would if be safer to pass the admin feedback form as an arg to this mutation instead? And the parent component should be rendering loading state if feedbackform has not been fetched.

Reply:
point taken, thanks! will fix